Rakefile

```diff
@@ -54,33 +54,39 @@ desc "Runs all linters. Run `rake -D lint` to see all available lint options"
 task lint: ["lint:lint"]
 
 namespace :docker do
+  def docker_sh(cmd)
+    docker_init = "eval \"$(docker-machine env default)\";"
+    sh "#{docker_init} #{cmd}"
+  end
+
   desc "Run Rubocop linter from docker"
   task :rubocop do
-    sh "docker-compose run lint rake lint:rubocop"
+    docker_sh "#{docker_init} docker-compose run lint rake lint:rubocop"
```

Comment:
@justin808 I think we will have problems with this. 
1. certain people might name their docker-machine instance something different than `default`. Like `dev` for example. Hardcoding the machine name here will cause an error.

2. Linux users will not be using docker-machine or boot2docker as a  lightweight linux vm to run docker in since it runs native on linux. This is only needed for mac and windows users.  'docker_init' will cause rake to abort for linux users.

I think it is better for users to place `eval "$(docker-machine env default)"` in their .zshrc or .bashrc files. or manually call `eval "$(docker-machine env default)"`if they need to execute.